docs/docs/references/users/README.md

```diff
@@ -98,19 +98,19 @@ _username_ is used for sign-in with _username_ and password.
 When the user has not registered with their username, their _username_ may be empty.
 When the _username_ is non-empty, it should be at most 128 characters, only contain letters, numbers, and underscores (`_`), and NOT start with a number.
 
-#### primary email
+#### primary_email
```

Comment:
should we mention that primary email/phone is verified?
it should not appear in these two fields if a phone number or email is not verified.

Reply:
Per Slack thread [discussion](https://silverhand-io.slack.com/archives/C02CMB8JVJB/p1656919713519819?thread_ts=1656835704.413199&cid=C02CMB8JVJB).